FWCore/Framework/test/eventsetup_t.cppunit.cc

```diff
@@ -590,77 +590,145 @@ namespace {
     ESGetToken<edm::eventsetup::test::DummyData, edm::DefaultRecord> m_token;
   };
 
-  class ConsumesProducer : public ESProducer {
-  public:
-    ConsumesProducer() : token_{setWhatProduced(this, "consumes").consumes<edm::eventsetup::test::DummyData>()} {}
-    std::unique_ptr<edm::eventsetup::test::DummyData> produce(const DummyRecord& iRecord) {
-      auto const& data = iRecord.get(token_);
-      return std::make_unique<edm::eventsetup::test::DummyData>(data);
-    }
+  //This just tests that the constructs will properly compile
+  class [[maybe_unused]] EDConsumesCollectorConsumer
+      : public edm::EDConsumerBase{EDConsumesCollectorConsumer(){using edm::eventsetup::test::DummyData;
```

Comment:
The indentation looks confusing.

Reply:
That was from clang format

Reply:
I wonder what confuses it (maybe the attribute?). Anyway, we can deal with that later.